### PR TITLE
Study Hub QA - 14 Dec

### PR DESCRIPTION
**Pull request by juliatan**
# Description

Notion tickets:

- https://www.notion.so/joinzoe/Dementia-Study-typo-1a88c4c83b3840c58c40399102646c8e
- https://www.notion.so/joinzoe/Studies-page-colour-bars-should-100-opacity-at-all-times-4daecc8a895941b29b8fdb9f77e2920b
- https://www.notion.so/joinzoe/In-page-scroll-vs-returning-from-Home-9093fca35a524757b54b9081c54f1843
- https://www.notion.so/joinzoe/Screen-2-Uneven-titles-Immune-health-Habits-behaviours-Future-disease-risk-all-have-differ-f64ef2a4f23d4cefb5f2ede08fb3b5ef
- https://www.notion.so/joinzoe/Study-Details-Page-Minimise-vertical-spacing-header-0d5b9175b5a048e1ba60277c5725d9ce
- https://www.notion.so/joinzoe/Overlay-copy-too-close-to-border-of-circle-bbeefd2e8db147dfa1cc65afc0b5401f

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Out of scope and potential follow-up

[comment]: <> (Are there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?)

## Screenshots

[comment]: <> (Please attach screenshots showing the change, if relevant)

## Checklist (for submission)

- [ ] Analytics/tracking events have been added (if relevant)
- [ ] Added new screen(s) to `src/features/screens.ts` and new Redux slice(s) to `mocks/mockedInitialState.ts` (if relevant)

## Checklist (for reviewers)

- [x] Checked against Figma designs (if relevant)
- [ ] Checked that data has been successfully saved to the backend (if relevant)

## Checklist (for merger)

- [ ] Make sure the right merge strategy is chosen, commit merge into master and squash merge into develop.

### CODE REVIEW

File: src/features/study-hub/screens/StudiesListScreen.tsx [hunk 1]
```diff
@@ -50,6 +51,12 @@ export function StudiesListScreen() {
     React.useCallback(() => {
       let isActive = true;
 
+      // Product requested we scroll to top of Studies list on screen focus
```
Comment (bionara) [inline src/features/study-hub/screens/StudiesListScreen.tsx:54]: is it annoying or useful for the user? :D

File: ios/Podfile.lock [hunk 2]
```diff
@@ -736,4 +736,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 0f7492c20a820386e7bf69b030b9c445e5f2a91b
 
-COCOAPODS: 1.10.2
+COCOAPODS: 1.11.2
```
Comment (bionara) [inline ios/Podfile.lock:739]: uh oh... is our build going to like this... stay tuned?! :D

**Review by bionara — APPROVED** [review]
Looks good. A little concerned our fragile build may be upset at the pod ver update:
COCOAPODS: 1.10.2
COCOAPODS: 1.11.2
but keen to merge, test, and fix if any issues.

File: src/features/study-hub/screens/StudiesListScreen.tsx (hunk 1 above)
Comment (juliatan) [reply src/features/study-hub/screens/StudiesListScreen.tsx:54]: There's limited scrolling on the studies list screen anyway, so in the balance, I think it makes sense.

**Pull request merged**